pytorch_lightning/trainer/trainer.py

```diff
@@ -199,6 +197,7 @@ def __init__(
         terminate_on_nan: bool = False,
         auto_scale_batch_size: Union[str, bool] = False,
         prepare_data_per_node: bool = True,
+        amp_type: str = 'native',
```

Comment:
was this the default before? let's make sure it's correct in the docs.

Reply:
before it uses native if available and if not, try apex... not you can choose on your own

Reply:
got it!!